frame/support/procedural/src/lib.rs

```diff
@@ -280,12 +284,78 @@ pub fn decl_storage(input: TokenStream) -> TokenStream {
 /// - `Inherent` - If the module provides/can check inherents.
 /// - `ValidateUnsigned` - If the module validates unsigned extrinsics.
 ///
+/// if no module parts is provided construct_runtime will try to use the expansion provided by the
+/// crate through `decl_construct_runtime_args`.
+///
 /// # Note
 ///
 /// The population of the genesis storage depends on the order of modules. So, if one of your
 /// modules depends on another module, the module that is depended upon needs to come before
 /// the module depending on it.
+/// ```
 #[proc_macro]
 pub fn construct_runtime(input: TokenStream) -> TokenStream {
 	construct_runtime::construct_runtime(input)
 }
+
+/// Macro than expand after the first finding of some pattern.
+///
+/// # Example:
+///
+/// ```nocompile
+/// expand_after!(
+///     { match pattern } // the match pattern cannot contains any group: `[]`, `()`, `{}`.
+///     { expansion tokens } // content inside braces can be anything including groups.
+///     Some content with { at some point match pattern } other match pattern are ignored
+/// );
+/// ```
+///
+/// will generate:
+///
+/// ```nocompile
+///     Some content with { at some point something or else } other match pattern are ignored
```

Comment:
I don't get this example. Should it not replace `match pattern` with `expansion tokens`? 

Like: 

```suggestion
///     Some content with { at some point expansion tokens } other match pattern are ignored
```

Reply:
I forget to udpate this example, it should be:
```
    Some content with { at some point match pattern expansion tokens } other match pattern are ignored
```
thanks